tests/php/core-api/wpcom-endpoints/test_class-wpcom-rest-api-v2-endpoint-external-media.php.php

```diff
@@ -0,0 +1,249 @@
+<?php // phpcs:ignore
```

Comment:
Minor: I assume it's unintentional that this file ends in `.php.php`

Reply:
Nice catch! Fixed in 3b4a5f5.